### PR TITLE
Addons: `load_when_embedded` config

### PR DESCRIPTION
**Pull request by humitos**
Add an extra addons config to decide whether or not force the injection of addons when the page is embedded (eg. iframe). By default, we are not loading addons if embedded.

Required by https://github.com/readthedocs/addons/pull/415
Closes https://github.com/readthedocs/addons/issues/412

### CODE REVIEW

File: readthedocs/proxito/views/hosting.py [hunk 1]
```diff
@@ -454,6 +454,9 @@ def _v1(self, project, version, build, filename, url, request):
             # Mainly, all the fields including a Project, Version or Build will use the exact same
             # serializer than the keys ``project``, ``version`` and ``build`` from the top level.
             "addons": {
+                "configs": {
```
Comment (agjohnson) [inline readthedocs/proxito/views/hosting.py:457]: Is `configs` new? If so, I'd use either `config` or `options` instead. I wouldn't normally use the word `configs` to describe multiple configuration options.

**Review by agjohnson — APPROVED** [review]
Looks good! Just a note on naming.

File: readthedocs/proxito/views/hosting.py (hunk 1 above)
Comment (humitos) [reply readthedocs/proxito/views/hosting.py:457]: Yes, it's new.

I'm thinking of this field as "configurations for addons in general that don't belong to a particular addon". `config` or `options` work fine to me as well 👍🏼

Comment (humitos) [reply readthedocs/proxito/views/hosting.py]: I went ahead and used `options`.

File: readthedocs/proxito/tests/responses/v1.json [hunk 2]
```diff
@@ -121,6 +121,9 @@
     }
   },
   "addons": {
+    "configs": {
+      "load_when_embedded": false
+    },
```
Comment (stsewd) [inline readthedocs/proxito/tests/responses/v1.json:126]: .com tests will probably need to be updated

Comment (humitos) [reply readthedocs/proxito/tests/responses/v1.json]: Hrm, I don't think we have this file in .com. At least, I didn't find it

Comment (stsewd) [reply readthedocs/proxito/tests/responses/v1.json]: We have two files in tests/responses/addons/

**Pull request merged**